lib/workers/pr/changelog/index.ts

```diff
@@ -28,9 +29,9 @@ export async function getChangeLogJSON(
     let res: ChangeLogResult | null = null;
 
     if (
-      args.sourceUrl?.includes('gitlab') ||
-      (args.platform === 'gitlab' &&
-        new URL(args.sourceUrl).hostname === new URL(args.endpoint).hostname)
+      (args.platform === PLATFORM_TYPE_GITLAB &&
+        new URL(args.sourceUrl).hostname === new URL(args.endpoint).hostname) ||
```

Comment:
`new URL()` throws if its param is not a valid URL. Should we make a util function in `lib/util/url.ts` that does a try/catch and then returns `null` or perhaps `{}`, or can we be certain that `sourceUrl` will always be defined and valid at this point in the code?

Reply:
@rarkins: I fixed this, please review again